src/roles/succubus.py

```diff
@@ -0,0 +1,315 @@
+            for e in ENTRANCED:
+                # to ensure we do not double-kill someone, notify all child deaths that we'll be
+                # killing off everyone else that is entranced so they don't need to bother
+                dlc = list(evt.params.deadlist)
+                dlc.extend(ENTRANCED - {e})
+                debuglog("{0} ({1}) SUCCUBUS DEATH KILL: {2} ({3})".format(nick, nickrole, e, get_role(e)))
+                evt.params.del_player(cli, e, end_game=False, killer_role="succubus",
+                    deadlist=dlc, original=evt.params.original, ismain=False)
+                evt.data["pl"] = evt.params.refresh_pl(evt.data["pl"])
+        ENTRANCED_DYING.clear()
+
+@event_listener("transition_day_resolve", priority=1)
+def on_transition_day_resolve(evt, cli, var, victim):
+    if victim in var.ROLES["succubus"] and VISITED.get(victim) and victim not in evt.data["dead"] and victim in evt.data["onlybywolves"]:
+        # TODO: check if this is necessary for succubus, it's to prevent a message playing if alpha bites
+        # a harlot that is visiting a wolf, since the bite succeeds in that case.
+        if victim not in evt.data["bitten"]:
+            evt.data["message"].append(messages["target_not_home"])
+            evt.data["novictmsg"] = False
+        evt.stop_processing = True
+        evt.prevent_default = True
+
+@event_listener("transition_day_resolve_end", priority=1)
+def on_transition_day_resolve_end(evt, cli, var, victims):
+    for victim in victims + evt.data["bitten"]:
+        if victim in evt.data["dead"] and victim in VISITED.values() and (victim in evt.data["bywolves"] or victim in evt.data["bitten"]):
+            for succ in VISITED.keys():
+                if VISITED[succ] == victim and succ not in evt.data["bitten"] and succ not in evt.data["dead"]:
+                    if var.ROLE_REVEAL in ("on", "team"):
+                        evt.data["message"].append(messages["visited_victim"].format(succ, get_role(succ)))
+                    else:
+                        evt.data["message"].append(messages["visited_victim_noreveal"].format(succ))
+                    evt.data["bywolves"].add(succ)
+                    evt.data["onlybywolves"].add(succ)
+                    evt.data["dead"].append(succ)
+
+@event_listener("night_acted")
+def on_night_acted(evt, cli, var, nick, sender):
+    if VISITED.get(nick):
+        evt.data["acted"] = True
+
+@event_listener("chk_nightdone")
+def on_chk_nightdone(evt, cli, var):
+    evt.data["actedcount"] += len(VISITED)
+    evt.data["nightroles"].extend(var.ROLES["succubus"])
+
+@event_listener("targeted_command")
+def on_targeted_command(evt, cli, var, cmd, actor, orig_target, tags):
+    if "beneficial" not in tags and actor in ENTRANCED and evt.data["target"] in var.ROLES["succubus"]:
+        try:
+            what = evt.params.action
+        except AttributeError:
+            what = cmd
```

Comment:
Please use a better variable name than `cmd` or `command`, to avoid confusion with the decorators.